python/pyarrow/_exec_plan.pyx

```diff
@@ -61,10 +65,18 @@ cdef execplan(inputs, output_type, vector[CDeclaration] plan, c_bool use_threads
         CTable* c_table
         shared_ptr[CTable] c_out_table
         shared_ptr[CSourceNodeOptions] c_sourceopts
+        shared_ptr[CScanNodeOptions] c_scanopts
+        shared_ptr[CExecNodeOptions] c_input_node_opts
         shared_ptr[CSinkNodeOptions] c_sinkopts
         shared_ptr[CAsyncExecBatchGenerator] c_async_exec_batch_gen
         shared_ptr[CRecordBatchReader] c_recordbatchreader
         vector[CDeclaration].iterator plan_iter
+        vector[CDeclaration.Input] no_c_inputs
```

Comment:
What does the "no_" stand for? (and what's the reason it was added, as it doesn't seem to be actually used except passed uninitialized?)

Reply:
it's a variable that contains no `Input`s, it's used to create the first node of the exec plan as it doesn't receive any other input, being it the first one.

Reply:
OK, I see, I found the naming a bit confusing at first, but if you read it correctly it's clear ;)